ReadMe.md

```diff
@@ -8,6 +8,9 @@
 [![Forks](https://img.shields.io/github/forks/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
 [![Stars](https://img.shields.io/github/stars/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
 [![Watchers](https://img.shields.io/github/watchers/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd)
+ 
+ 
+[![DevTernShip Introduction](https://i.postimg.cc/9M4mCZj4/efs.png)](https://postimg.cc/BL09C8bs.jpg)
```

Comment:
Why are we using some crazy service like postimg? Already the image URL is given right? Use the same. And maybe link it to the video rather than the PostIMG website. We're not advertising for them! 😅

Reply:
Oh, Sorry. My bad. I'll change it.